docs/DeveloperGuide.md

```diff
@@ -383,6 +383,29 @@ Priorities: Essential (must have) - `* * *`, Typical (nice to have) - `* *`, Nov
 
       Use case ends.
 
+**Use case: Searching for patients**
+
+**MSS**
+
+1. User requests to find patients from a keyword.
+2. System shows a list of patients that match the keyword with their names.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. System detects invalid request format
+
+    * 1a1. System shows an error message.
+
+      Use case ends.
+
+* 1b. System could not find the patient from the keyword
```

Comment:
Perhaps this could be `find any patients` instead of `find the patient` to match the statements in the MSS.